recipes/mender-monitor/debian-1.1.x/control

```diff
@@ -0,0 +1,19 @@
+Source: mender-monitor
+Maintainer: Mender Team <mender@northern.tech>
+Section: admin
+Priority: optional
+Standards-Version: 3.9.2
+Build-Depends: debhelper (>= 9)
+
+Package: mender-monitor
+Architecture: all
+Depends: bash, lmdb-utils, mender-auth | mender-client (>=2.5.0), ${misc:Depends}
+Description: Mender Monitor client
```

Comment:
I checked all the docs I could find, and we nowhere refer to "Mender Monitor client", we always say `Monitor add-on` hence my kind suggestion
```suggestion
Description: Mender Monitor add-on client
```